hieradata/node/atarchiver.cp.lsst.org.yaml

```diff
@@ -76,6 +76,25 @@ nfs::nfs_exports_global:
       139.229.160.0/24(ro,nohide,insecure,no_subtree_check,async,root_squash)
       139.229.165.0/24(ro,nohide,insecure,no_subtree_check,async,root_squash)
       139.229.175.128/25(ro,nohide,insecure,no_subtree_check,async,root_squash)
+  /data/repo:
+    clients: >-
+      %{facts.networking.ip}/32(ro,nohide,insecure,no_subtree_check,async,root_squash)
```

Comment:
This is for the self-mount?  Can it be rw instead, then?

Reply:
change applied